layouts/partials/footer.html

```diff
@@ -9,7 +9,7 @@
   <div class="container is-max-widescreen">
     <div id="footer-columns" class="columns">
       <div class="footer-logo-column">
-        <img id="footer-logo" src="{{ printf "/images/%s" $logo | relURL }}" alt="{{ $title }} logo. {{ $navbarLogo.altText }}">
+        <img id="footer-logo" src="{{ printf "/images/%s" $logo | relURL }}" alt="{{ $title }} logo. {{ $navbarLogo.altText }}" width="67" height="75">
```

Comment:
Probably best to use a scalable unit like `rem`.

Reply:
Agreed, however HTML IMG tags specify sizes only in pixels.  The Lighthouse report recommended this change, which is the only reason for it.  But I'll see if it will accept specifying the size with a STYLE attribute in relative units.

Reply:
Further research shows that, according to the resources linked by the Lighthouse report, the current best practice is to specify the image's expected width and height in the HTML tag in pixels, mainly for the sake of indicating the image's aspect ratio, and then using CSS properties to set the image's final size after loading.  (See, e.g. [this web.dev article](https://web.dev/articles/optimize-cls?utm_source=lighthouse&utm_medium=node#images_without_dimensions).